src/pages/components/radio.mdx

```diff
@@ -1,10 +1,106 @@
 ---
 title: Radio
 description: >
-  Description will be here
+  Radio buttons let users choose a single option among two or more mutually exclusive options.
+package: '@zendeskgarden/react-forms'
+propSheets:
+  - forms/src/elements/Radio.tsx
+  - forms/src/elements/common/Field.tsx
+  - forms/src/elements/common/Hint.tsx
+  - forms/src/elements/common/Label.tsx
+  - forms/src/elements/common/Message.tsx
 ---
 
-Content here
+<Usage>
+  <Use>
+    <ul>
+      <li>For choices or options that can't occur at the same time.</li>
+      <li>To indicate that two or more options are mutually exclusive.</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>If the user can choose more than one option at once, use a Checkbox instead.</li>
+      <li>To select from a long list of options, use a Select.</li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+A Radio’s label is part of its touch target.
+
+import Default from '../../examples/forms/radio/Default.tsx';
+import DefaultCode from '!!raw-loader!../../examples/forms/radio/Default.tsx';
+
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
+
+### Regular weight label
+
+Radio labels are bold by default but can be regular weight.
+
+import RegularWeightLabel from '../../examples/forms/radio/RegularWeightLabel.tsx';
+import RegularWeightLabelCode from '!!raw-loader!../../examples/forms/radio/RegularWeightLabel.tsx';
+
+<CodeExample code={RegularWeightLabelCode}>
+  <RegularWeightLabel />
+</CodeExample>
+
+### Hint text
+
+Hint text adds additional context to a Radio label.
+
+import Hint from '../../examples/forms/radio/Hint.tsx';
+import HintCode from '!!raw-loader!../../examples/forms/radio/Hint.tsx';
+
+<CodeExample code={HintCode}>
+  <Hint />
+</CodeExample>
+
+### Validation
+
+Success, warning, or danger validation messages are shown using the Message component.
+
+import Validation from '../../examples/forms/radio/Validation.tsx';
+import ValidationCode from '!!raw-loader!../../examples/forms/radio/Validation.tsx';
+
+<CodeExample code={ValidationCode}>
+  <Validation />
+</CodeExample>
+
+### Hidden
+
+Radio labels can be removed.
```

Comment:
Should this sentence contain more words that help the user understand when to use hidden labels?

Reply:
Possibly, but I think for right now this works as is.